app/controllers/auth/google_controller.rb

```diff
@@ -68,8 +68,7 @@ def register_with_google(name, email, role, access_token, google_id)
     email = email.downcase
     if current_user && current_user.email.downcase != email
       session[:google_email] = email
-      redirect_to "/auth/google_email_mismatch/"
-      return
+      return redirect_to "/auth/google_email_mismatch/"
```

Comment:
I understand why having a return after a redirect_to is probably pointless, but why return the redirect?

Reply:
Having the return on the same line as the redirect is just terser and more Rubyish. Also, keeping them together helps ensure we don't accidentally try to do some stuff afterward that results in a double render or double redirect error.